### ISSUE TITLE
Too many arguments for MappingView

### ISSUE THREAD

**Issue by gsakkis** [python/typeshed#3549]
The following snippet fails with `Too many arguments for "ValuesView"  [call-arg]` although that's the correct signature of `ValuesView` (and all mapping views for that matter).
```
from typing import ValuesView

v: ValuesView[int] = ValuesView({"foo": 1})
print(list(v))
```
Versions:
- Python 3.6.8
- mypy 0.760

**Comment by srittau** [comment]
`ValuesView` (or rather `MappingView`) does not have an `__init__` method in typeshed. PR welcome!

**Comment by hauntsaninja** [comment]
This should be fixed by #3528, although that doesn't make the change for Python 2 (where MappingView can't be instantiated, but the others can).
